pkg/controller/pod/pod_controller.go

```diff
@@ -221,53 +220,40 @@ func (r *ReconcilePod) Reconcile(request reconcile.Request) (reconcile.Result, e
 }
 
 func (r *ReconcilePod) recreateNsxNcpResourceIfDeleted(resName string) error {
-	instance := identifyAndGetInstance(resName)
-	instanceDetails := types.NamespacedName{
-		Namespace: operatortypes.NsxNamespace,
-		Name:      resName,
-	}
-
-	doesResExist, err := r.checkIfK8sResourceExists(instance, instanceDetails)
+	doesResExist, err := operatortypes.CheckIfNCPK8sResourceExists(
+		r.client, resName)
 	if err != nil {
 		log.Error(err, fmt.Sprintf(
-			"Could not retrieve K8s resource - '%s'", instanceDetails.Name))
+			"Could not retrieve K8s resource - '%s'", resName))
 		return err
 	}
 	if doesResExist {
 		log.Info(fmt.Sprintf(
-			"K8s resource - '%s' already exists", instanceDetails.Name))
+			"K8s resource - '%s' already exists", resName))
```

Comment:
more nits! This could go down to debug leve

Reply:
updated to `log.V(1)`